src/tests/test_addressbook.py

```diff
@@ -0,0 +1,20 @@
+"""
+Tests for addresbook validation
+"""
+
+import unittest
+from pybitmessage.bmconfigparser import BMConfigParser
+
+
+class TestAddAddressBook(unittest.TestCase):
+    """this class is used for testting add address feature"""
+
+    def test_is_own_address_add_to_addressbook(self):
+        """Check the logic of TCPConnection.local"""
+        own_addresses = BMConfigParser().addresses()
```

Comment:
call the helper function twice with the same address, first it should succeed, then fail.